jwst/resample/resample_step.py

```diff
@@ -68,7 +68,7 @@ def process(self, input):
             self.log.info('Drizpars reference file: {}'.format(ref_filename))
             kwargs = self.get_drizpars(ref_filename, input_models)
         else:
-            # Deal with NIRSpec which currently has no default drizpars reffile
+            # Deal with FGS which currently has no default drizpars reffile
```

Comment:
Why does the comment here say "FGS", but the log message in the next line says "NIRSpec"? It would be better to stay instrument-agnostic and just say there's no ref file, period, with no mention of instruments.

Reply:
Doh!  Didn't catch the log message.  Agree it should be instrument-agnostic.  And once we move to `pars-resamplestep` CRDS reffiles, we should just remove all of this (and the `drizpars` CRDS reffile) to boot.  A future PR.